apps/dashboard/mixins.py

```diff
@@ -123,6 +123,27 @@ def post(self, request, *args, **kwargs):
                         organisation_slug=self.organisation.slug)
 
 
+class DashboardUserRemovalMixin:
```

Comment:
oh actually i just forgot to delete the mixins, they have both been moved to views.py as 
DashboardProjectUserListView and DashboardProjectModeratorsView